### ISSUE TITLE
View the failed rows from a reference check

### ISSUE THREAD

**Issue by pwrignall** [sodadata/soda-core#1439]
For this check:
```
checks for LOCATIONS:
  - values in (LOC_TYPE_CODE) must exist in LOCATION_TYPES (LOC_TYPE_CODE)
```
This is the result I see. Is there a way to see those 3 failed records (or at least the invalid LOC_TYPE_CODE values) in Soda Cloud, like we can with other checks?

![image](https://user-images.githubusercontent.com/8555469/175069560-5683289d-96ac-4a16-9dd3-b58a03a6163f.png)

(from https://soda-community.slack.com/archives/C01TFMC2S57/p1655910623768949)

**Comment by jmarien** [comment]
SODA-909

**Comment by vijaykiran** [comment]
Fixed in 3.0.3

**Comment by vijaykiran** [comment]
@pwrignall  Can you please check with 3.0.3 and let us know if it works?
